### DISCUSSION TITLE
Please take the Pre-Course Survey

### DISCUSSION THREAD

**Discussion by nquarder** [earthlab-education/Earth-Analytics-AY24#4] [Announcements]
We would love to collect some information from each of you regarding your backgrounds and prior experiences. Please take 10-15 minutes to complete the Pre-Course Survey. Your feedback will help us make this program better. Please let me know if you have any questions or concerns about this survey. -Nate Q

**https://cuboulder.qualtrics.com/jfe/form/SV_6M8yLrFSHYjMGs6**

**Comment by BioNCphilic** [comment]
There were two questions on the pre-course survey that i could not answer as they had to do with the pace of the course and the workload. Hard to assess that before starting the course!

**Reply by akpost21** [reply]
@nquarder Just checking that this was the correct link for the pre-course survey?

**Reply by nquarder** [reply]
Hi @akpost21 and @BioNCphilic - good catch on this! Yes, it is the correct survey however we forgot to remove a couple of unwanted items. Just answer those items asking about your experience so far as best you can. We'll remove those from our analysis and will reach out again around mid-term and again after the semester to ask those questions about pace of course, time spent on assignments, etc. Again, thanks for catching this y'all! -NQ

**Reply by akpost21** [reply]
@nquarder wonderful, thank you!
